lib/assessment.rb

```diff
@@ -20,20 +27,44 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+
+  attr_writer :name, :location
+  attr_reader :age
+
+  def initialize (name, age, location)
+    @name, @age, @location = name, age, location
+  end
+end
 ##
 
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
 
-Answer.dave = nil # replace nil with your code
-Answer.dave.location = nil # replace nil with your code
+Answer.dave = Person.new("Dave", 32, "Ohio")
+Answer.dave.location = "Somerville"
 
 # Question 5
 # Create another class called Developer that inherits from Person.
 # Give it a new public method called 'hire_for_job'
 
 ##
-# your code here
+class Developer < Person
+  def initialize (name, age, location)
+    @name, @age, @location = name, age, location
+  end
```

Comment:
This is actually not necessary - `Developer` inherits from `Person`, so it is able to run all of the methods that `Person` can (unless we specifically choose to overwrite them).